CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/dragdrop/JsonDropper.java

```diff
@@ -86,10 +86,10 @@
         return null;
     }
 
-    @PluginInfo(pluginType = PluginType.UPDATE, tags = {"MODIFY"})
+    @PluginInfo(pluginType = PluginType.IMPORT, tags = {"LOW LEVEL"})
     public static class DragAndDropJSONPlugin extends SimpleEditPlugin {
```

Comment:
I think this function is actually adding to an existing graph (rather than opening a graph like the other function) so what you tagged it with previously is ok

Reply:
Similarly, I would suggest `IMPORT` as otherwise you have both an `IMPORT` and `MODIFY` tags effectively for this action.

Reply:
Sorry do you mean ` PluginType.IMPORT, tags = {"IMPORT"}`?

Reply:
Yeah @Auriga2